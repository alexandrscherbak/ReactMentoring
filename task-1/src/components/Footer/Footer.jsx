import React from 'react';
import Logo from '../Logo/Logo.jsx'

const Footer = () => (
	<div>
		<Logo />

		<style jsx>
			{`
				div {
					width: calc(100% - 30px);
					height: 50px;
					background-color: #414141;
					padding-left: 30px;
					line-height: 50px;
				}
			`}
		</style>
	</div>
);

export default Footer;
