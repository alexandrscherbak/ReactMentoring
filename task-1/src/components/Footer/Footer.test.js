import React from 'react';
import Footer from './Footer.jsx';
import renderer from 'react-test-renderer';

describe('Footer', () => {
	test('snapshot', () => {
		const component = renderer.create(
			<Footer/>,
		);
		let tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
