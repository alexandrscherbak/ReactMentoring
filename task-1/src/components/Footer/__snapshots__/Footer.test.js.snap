// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot 1`] = `
<div
  className="jsx-429413428"
>
  <div
    className="jsx-2852891300"
  >
    <span
      className="jsx-2852891300"
    >
      netflixroulette
    </span>
  </div>
</div>
`;
